apps/staking/lib/locale-client.ts

```diff
@@ -42,8 +42,7 @@ export const formatLocalizedTimeFromSeconds = (
   formatLocalizedRelativeTimeClient(getDateFromUnixTimestampSeconds(seconds), new Date(0), options);
 
 export const formatNumber = (num: number, options?: Intl.NumberFormatOptions) => {
-  const locale = useLocale();
-  return new Intl.NumberFormat(locale, options).format(num);
+  return new Intl.NumberFormat(undefined, options).format(num);
```

Comment:
No need for locale anymore?

Reply:
this was a problem, system locale is default in Intl, so we were making it into a hook for no reason and causing headaches

Reply:
Not ideal but it is what it is. Resolving